Deployment/Mirantis-Guidance.md

```diff
@@ -26,16 +26,24 @@ Customer is using Windows Server image 2019 with Containers
 
 ## Migration risk decision guide
 
+The possible options for the migration are combinations of the following controls:
+   - OS SKU selection
+   - container runtime selection (MCR, Moby, other or none)
+   - container runtime installation mechanism 
+   - cluster nodes update mechanism (in place, new node types or recreating the cluster)
+
 This guide is designed to help you assess the effort and risk of each migration option. 
-Criteria for successfully running MCR to host container on Azure Service Fabric cluster.
+<!-- Criteria for successfully running MCR to host container on Azure Service Fabric cluster.
 1. Acquire a Mirantis Container Runtime license.
 2. Service Fabric runtime needs to be on version [9.0 CU2 (9.0.1048.9590) or greater](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-versions).
 3. Azure Virtual Machine Scale Sets to host containers on MCR needs to run on Windows Server 2022. 
-4. MCR needs to be installed by Custom Script VM Extension or pre-installed as part of an OS image.
+4. MCR needs to be installed by Custom Script VM Extension or pre-installed as part of an OS image. -->
 
 In-place SKU upgrades are in general not supported on Service Fabric cluster nodes, as such operations potentially involve data and availability loss. The safest, most reliable, and recommended method for scaling up a Service Fabric node type is to add a new node type and move the workload over.
 
-In this document we are describing an approach to do an in-place OS SKU upgrade with less effort but the potential risk of ending up in a non-recoverable state. This approach should be only considered for Service Fabric Node Types without container workloads. Please read the described risks for each scenario carefully.
+<!-- In this document we are describing an approach to do an in-place OS SKU upgrade with less effort but the potential risk of ending up in a non-recoverable state. This approach should be only considered for Service Fabric Node Types without container workloads. Please read the described risks for each scenario carefully. -->
```

Comment:
>potential risk of ending up in a non-recoverable state. [](http://example.com/codeflow?start=110&length=55)

I commented this out as the in-place is one of the option. Also, we should not willingly/openly suggest to customers a faster migration that could leave them in a non-recoverable state.

It is fine to say that is a possible choice, but comes with a risk. Note cx without containers aren't necessarily more tolerant of non-recoverable failed upgrades.